### DISCUSSION TITLE
Wizznet (W5100, W5500, W6100) net chips support

### DISCUSSION THREAD

**Discussion by xRadx** [stm32duino/Arduino_Core_STM32#2617] [Q&A]
Hi,

I have few Nucleo f303k8 and tried many libraries (including generic Ethernet and STM32Ethernet) for these chipsets, but none seems to work (with nucleos + wizznet chipsets).

it there planed support for them?

**Comment by fpistm** [comment]
This one should work:
https://github.com/arduino-libraries/Ethernet/releases

**Comment by xRadx** [comment]
Thanks for your reply. Unfortunately for me it does not work...
I am getting an error:
"Ethernet shield was not found.  Sorry, can't run without hardware. :("

Nucleo is connected to wiz810sio pins as follows:
13 - SC(L)K
12 - MOSI
11 - MISO
10 - SC (SS)

The same module connected to the same pins of Arduino Nano is working fine.

**Reply by fpistm** [reply]
Hi @xRadx 
looking at your module it uses the W5100S and it is not yet supported by the Arduino Ethernet library, see https://github.com/arduino-libraries/Ethernet/issues/207
You can try the PR:
https://github.com/arduino-libraries/Ethernet/pull/185

**Comment by xRadx** [comment]
I've overwritten the files in my Arduino library (also checked these were used), but still no success - No hardware found.

**Reply by fpistm** [reply]
OK. Unfortunately, I will not be able to help more as I do not have this hardware.
Ensure your hardware is  properly wired and check the voltage level, Arduino board is mainly 5V while stm32 is 3.3V.
